src/utils.py

```diff
@@ -1987,7 +1987,11 @@ def get_acroform(doc):
                 w_obj.pdf_dict_get(pymupdf.PDF_NAME("Parent"))
             )
             if parent_xref == 0:  # parent not in target yet
-                w_obj_graft = mupdf.pdf_graft_mapped_object(graftmap, w_obj)
+                try:
+                    w_obj_graft = mupdf.pdf_graft_mapped_object(graftmap, w_obj)
+                except Exception as e:
```

Comment:
Is there a specific exception type that we could catch here so that we don't hide unexpected errors?